setup.py

```diff
@@ -38,7 +38,7 @@
     python_requires="~=3.6",
     install_requires=["django>=2.2", "django-ipware>=3,<4"],
     include_package_data=True,
-    packages=find_packages(),
+    packages=find_packages(include=["axes"]),
```

Comment:
Have you confirmed that this will include the subpackages under `axes/` in the sdist and wheel?

Reply:
Yes, `python setup.py sdist` and `python setup.py bdist_wheel` do appear to work correctly with this definition and do include e.g. `axes.handlers` and other submodules in the package sources.

Reply:
Ended up using `exclude=["tests"]` which seems to do the trick. All other Python modules are included as default.